transmission_interface/src/transmission_parser.cpp

```diff
@@ -12,68 +12,250 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "transmission_interface/transmission_parser.hpp"
-#include "transmission_interface/transmission_info.hpp"
+#include <transmission_interface/transmission_parser.hpp>
 
-#include <tinyxml2.h>
-#include <stdexcept>
+// ROS
+#include <rclcpp/rclcpp.hpp>
+
+#include <sstream>
 #include <string>
 #include <vector>
```

Comment:
```suggestion
#include <sstream>
#include <string>
#include <vector>

#include <rclcpp/rclcpp.hpp>
```

Was this accepted by the linters? ROS dependency should be after C++ dependencies.

Reply:
yes, it was accepted by ament_cpplint with no output. Making the correction though.

Reply:
Actually, it complains about headers if I make the move of rclcpp down:
`/home/guru/src/ros/ros2_control/transmission_interface/src/transmission_parser.cpp:21:  Found C system header after C++ system header. Should be: transmission_parser.h, c system, c++ system, other.  [build/include_order] [4]
`
It appears to be detecting rclcpp/rclcpp.hpp as a C header.

Reply:
This is strange. Is it maybe the issue with `sstream`?

Reply:
Switching to quotes satisfied the linter but not sure I agree with that being the correct solution. quotes typically mean the file is within the module and rclcpp is not. strange. I will try to get rid of the rclcpp dependency anyway so this maybe moot.